### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by rstz**

### CODE REVIEW

File: pthreadfs/README.md [hunk 1]
```diff
@@ -71,8 +65,9 @@ See `pthreadfs/examples/emscripten-tests/fsafs.cpp` for exemplary usage.
 - The code is still prototype quality and **should not be used in a production environment** yet. It is possible that the use of PThreadFS might lead to subtle bugs in other libraries.
 - PThreadFS requires PROXY_TO_PTHREAD to be active. In particular, no system calls interacting with the file system should be called from the main thread.
 - Some functionality of the Emscripten File System API is missing, such as sockets, IndexedDB integration and support for XHRequests.
+- PThreadFS does not support symlinks.
+- Changing the current working directory might not work in some cases.
```
Comment (fivedots) [inline pthreadfs/README.md:69]: Is there more information on this e.g. how to avoid it? or is it an unknown bug for now?

**Review by fivedots — APPROVED** [review]
LGTM modulo comments.

File: pthreadfs/README.md (hunk 1 above)
Comment (rstz) [reply pthreadfs/README.md]: Done.

File: pthreadfs/README.md [hunk 2]
```diff
@@ -2,26 +2,19 @@
...
-
-If OPFS Access Handles are not available, PThreadFS will use an in-memory file system.
+PThreadFS replaces Emscripten's internal file system layer with a system that proxies all file system operations to a dedicated thread. PThreadFS then stores the file contents using a suitable Chrome storage API. In particular, PThreadFS comes with built-in support for [OPFS Access Handles](https://docs.google.com/document/d/121OZpRk7bKSF7qU3kQLqAEUVSNxqREnE98malHYwWec/edit#heading=h.gj2fudnvy982). If OPFS Access Handles are not available, PThreadFS will use an in-memory file system.
 
 The code is still prototype quality and **should not be used in a production environment** for the time being.
 
 ## Enable OPFS Access Handles in Chrome
 
-OPFS Access Handles require Google Chrome 96. "Experimental Web Platform Features" must be enabled in [chrome://flags](chrome://flags).
+OPFS Access Handles are available as [origin trial](https://developer.chrome.com/origintrials/) in Chrome 96.
```
[the remark this answers is not in the document]
Comment (rstz) [reply pthreadfs/README.md]: Done.

**Pull request merged**